project/app/views/list_items/_list_item.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="row list-item item-hide list<%= list_item.list_id%>">
```

Comment:
This bit:
`list<%= list_item.list_id%>"`
is where I'm talking about targeting for CSS with jquery